Classes/Extensions/DDDispatchQueueLogFormatter.m

```diff
@@ -93,48 +117,54 @@ - (void)setReplacementString:(NSString *)shortLabel forQueueLabel:(NSString *)lo
 #pragma mark DDLogFormatter
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-- (NSString *)stringFromDate:(NSDate *)date {
-    int32_t loggerCount = OSAtomicAdd32(1, &_atomicLoggerCount);
+- (NSDateFormatter *)createDateFormatter {
+    NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+    [self configureDateFormatter:formatter];
+    return formatter;
+}
```

Comment:
Just to be sure, we may need several instances of `formatter`? Asking just in case we could have a singleton.